src/views/HomePage.vue

```diff
@@ -4,34 +4,31 @@ import { defineComponent } from 'vue';
 import HeroSection from '@/components/HomePage/HeroSection/HeroSection.vue';
 import MoviesSection from '@/components/HomePage/MoviesSection/MoviesSection.vue';
 
+import { useMoviesStore } from '@/store/movies';
+const moviesStore = useMoviesStore();
```

Comment:
as it is not `script setup` composable should be called inside the `setup` function. Then you can return `movieStore` from this function and use it in the component scope 👌 
...or you can use `mapState` from Pinia instead

Reply:
Done!